distributed/bokeh/scheduler_html.py

```diff
@@ -163,6 +164,23 @@ def get(self):
         self.write(result)
 
 
+class PrometheusHandler(RequestHandler):
+    # Construct Prometheus metrics
+    # https://prometheus.io/docs/introduction/overview/
```

Comment:
If you'd like to provide a documentation string I recommend reading through http://docs.dask.org/en/latest/develop.html#docstrings